graphula-core/src/Graphula.hs

```diff
@@ -31,62 +30,82 @@
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE RecordWildCards #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TupleSections #-}
+{-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE TypeOperators #-}
 
 module Graphula
-  ( -- * Graph Declaration
-    node
-  , nodeEdit
-  , nodeWith
-  , nodeEditWith
+  ( -- ** Graph Declaration
```

Comment:
Oh, I didn't mean all these should be h2, just that the "Node Options" might be best placed under "Graph Declaration".

Reply:
Ah, I wasn't sure. I didn't know if Haddocks are supposed to obey the "only one H1 thing in the whole page" that applies to regular HTML